components/common/AppDrawer.tsx

```diff
@@ -28,6 +31,15 @@ const AppDrawer: StylableFC = ({ style, className }) => {
   const locale = useLocale();
   const { t } = useTranslation("common", { keyPrefix: "appDrawer" });
 
+  /**
+   * Track an app open event.
+   * 
+   * @param app The name of the app.
+   */
+  function trackAppOpen(app: string) {
```

Comment:
This is great for keeping the code short and organized because the opening of these apps must be applied to a range of elements.